src/main/kotlin/io/fabric8/launcher/booster/catalog/rhoar/RhoarBooster.kt

```diff
@@ -1,12 +1,9 @@
 package io.fabric8.launcher.booster.catalog.rhoar
 
-import java.util.Collections
-import java.util.HashMap
-import java.util.Objects
-import java.util.stream.Collectors
-
 import io.fabric8.launcher.booster.catalog.Booster
 import io.fabric8.launcher.booster.catalog.BoosterFetcher
+import java.util.*
```

Comment:
Can you configure your IDE to not use wildcards?

Reply:
Sure, do you have shared config at hand?

Reply:
Out of curiosity - what is the rationale behind that?

Reply:
Is that the desired one?

Reply:
Yes, that is what we use too